src/components/ViewRequest/ActionBar.tsx

```diff
@@ -0,0 +1,22 @@
+import SendRequest from "./Actions/SendRequest";
+import ReworkRequest from "./Actions/ReworkRequest";
+import EditRequest from "./Actions/EditRequest";
+import DeleteRequest from "./Actions/DeleteRequest";
+
+const ActionBar = () => {
+  return (
```

Comment:
Rather than hard coded line breaks, maybe wrap this is a column flexbox?  If you really want the linebreaks instead, I suggest at least cleaning up this so it is consistent with blank lines (or no blank lines) between each component.